lib/messages/release.js

```diff
@@ -0,0 +1,46 @@
+const moment = require('moment');
+const mrkdwn = require('html-to-mrkdwn');
+
+const { Message } = require('.');
+
+class Release extends Message {
+  constructor({
+    release, repository,
+  }) {
+    super({
+      footer: `<${repository.html_url}|${repository.full_name}`,
+    });
+    this.repository = repository;
+    this.release = release;
+  }
+
+  getRenderedMessage() {
+    const { release } = this;
+    const { author } = release;
+
+    const message = {
+      ...super.getBaseMessage(),
+      title: `Release - ${release.name}`,
```

Comment:
@wilhelmklopp I know we talked about ways of standardizing the "type" in messages, but I don't remember what we decided.

A few examples of what we do elsewhere:

- Comments: `Comment on #${this.issue.number} ${this.issue.title}`
- Issue/PR: `#${this.issue.number} ${this.issue.title}`
- Review: `Review on #${this.pull.number} ${this.pull.title}`
- Public: `${this.repository.full_name} is now public!`

Reply:
This message will also need a `fallback`, which can be the same as `title`, but prefixed with `[owner/repo]`.